web/src/lib/static/loadCertifications.ts

```diff
@@ -8,6 +7,7 @@ import { getFileNameByUrlSlug } from "./helpers";
 const certificationDir = path.join(process.cwd(), "..", "content", "src", "certifications");
 const archivedCertificationDir = path.join(process.cwd(), "..", "content", "src", "archived-certifications");
 
+type PathParams<P> = { params: P; locale?: string };
```

Comment:
So we re-define this like 7 times, is there a reason we don't have a unified type that we're exporting? (Or I assume there is a reason that I don't know that would be good if I did know)

Reply:
Well, I didn't like the old pattern, where randomly one file (fundings, I think) defined it and for some reason all these other files imported from fundings? That pattern breaks domain boundaries because loading X (quick actions, certs, etc) should certainly not depend on fundings.

Putting it in a top level import would work, but since we were already redefining it in some, I just leaned into that pattern for this refactor. Incremental cleanup, but I agree it could go further to refactor out the duplication altogether

Reply:
Gotcha